internal/database/models/models.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import (
+	"gorm.io/gorm"
+)
+
+type Author struct {
+	gorm.Model
+	FirstName     string `gorm:"not null"`
+	LastName      string
+	Bio           string  `gorm:"type:text"`
+	OriginCountry string  `gorm:"not null"`
+	CurrentCity   string  `gorm:"not null"`
+	Stories       []Story `gorm:"foreignKey:AuthorID;constraint:OnUpdate:CASCADE,OnDelete:CASCADE;"`
+}
+
+type Story struct {
+	gorm.Model
+	Title    string
+	Content  string `gorm:"type:text"`
+	AuthorID uint
```

Comment:
I see in the gorm docs that they use `int` in some examples, will `uint` be fine too?

Reply:
`AuthorID` is a the foreign key, so `uint` is more appropriate